modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -166,7 +166,25 @@ export class BaseAPI {
                 }) || fetchParams;
             }
         }
-        let response = await (this.configuration.fetchApi || fetch)(fetchParams.url, fetchParams.init);
+        let response = null;
+        try {
+            response = await (this.configuration.fetchApi || fetch)(fetchParams.url, fetchParams.init);
+        } catch (e) {
+            for (const middleware of this.middleware) {
+                if (middleware.onError) {
+                    response = await middleware.onError({
+                        fetch: this.fetchApi,
+                        url: fetchParams.url,
+                        init: fetchParams.init,
+                        error: e,
+                        response: response ? response.clone() : undefined,
+                    }) || response;
+                }
+            }
+            if (!response) {
+                throw new FetchError(e, 'Interceptors did not provide an alternative response');
```

Comment:
How about “The request failed and the interceptors did not return an alternative response?

Reply:
Sure, changed